code/game/machinery/atmoalter/canister.dm

```diff
@@ -32,9 +32,9 @@
 	melt_temperature = MELTPOINT_STEEL
 
 	//Icon Update Code
-	var/global/status_overlays = 0
-	var/global/list/status_overlays_pressure
-	var/global/list/status_overlays_other
+	var/status_overlays = 0
+	var/list/status_overlays_pressure
+	var/list/status_overlays_other
```

Comment:
Making it not global however caused the list to be duplicated across every canister object which is a waste of memory.

If you instead want to do something good like this I'd recommend you make a proc for this stuff with a static variable for the list, then override that on the subtype. See atmospherics notches and their overrides in heat exchangers (to allow a different notch for heat exchangers): https://github.com/d3athrow/vgstation13/blob/Bleeding-Edge/code/ATMOSPHERICS/atmospherics . dm https://github.com/d3athrow/vgstation13/blob/Bleeding-Edge/code/ATMOSPHERICS/components/unary/heat_exchanger.dm